app/src/main/java/com/example/jasmiensofiecels/wordly/view/Base/BaseActivity.java

```diff
@@ -0,0 +1,50 @@
+package com.example.jasmiensofiecels.wordly.view.Base;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.v7.app.AppCompatActivity;
+
+import com.example.jasmiensofiecels.wordly.R;
+import com.example.jasmiensofiecels.wordly.presenter.base.base.BaseView;
+
+import javax.annotation.Nullable;
+
+import dagger.android.AndroidInjection;
+
+/*
+ * The purpose of this class is to represent the underlying activity that all other activities will inherit from.
+ *
+ * Create by Jasmien Cels on 12/03/18
+ */
+
+public class BaseActivity extends AppCompatActivity implements BaseView {
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        AndroidInjection.inject(this);
+
```

Comment:
Left a comment here but it disappeared. Check out [`DaggerAppCompatActivity`](https://github.com/google/dagger/blob/master/java/dagger/android/support/DaggerAppCompatActivity.java), similar to above comment

Reply:
Do you mean DaggerActivity?

DaggerAppCompatActivity doesn't seem to be in version 2.11

Reply:
`compile 'com.google.dagger:dagger-android-support:2.x' // if you use the support libraries`

from [their readme](https://github.com/google/dagger).

[Why use the support libraries](https://developer.android.com/topic/libraries/support-library/index.html)? Using the support library can cause confusion later on (like when you have to use the `SupportFragmentManager` instead of `FragmentManager`) so if you're not using the support library already, I wouldn't worry. Just `DaggerActivity` should be good.